Makefile

```diff
@@ -79,14 +79,19 @@ resources:
 	target/release/parallel export-genesis-state --parachain-id 200 > ./resources/para-200-genesis
 	target/release/parallel export-genesis-wasm > ./resources/para-200.wasm
 
-.PHONY: register-parachain
-register-parachain:
-	polkadot-js-api --seed "//Alice" --sudo tx.parasSudoWrapper.sudoScheduleParaInitialize 200 "{ \"genesisHead\":\"$(shell cat ./resources/para-200-genesis)\", \"validationCode\":\"$(shell cat ./resources/para-200.wasm)\", \"parachain\": true }"
+.PHONY: docker-resources
+docker-resources:
+	docker run --rm  parallelfinance/parallel:latest parallel export-genesis-state --chain heiko-dev --parachain-id 200 > ./resources/para-200-genesis
```

Comment:
parachain-id should start from 2000, you can refer to https://substrate.dev/cumulus-workshop/#/en/2-relay-chain/2-reserve?id=reserve-a-paraid

Reply:
we haven't decided our paraid yet so 200 is only for testing